src/AMSlib/wf/workflow.hpp

```diff
@@ -52,18 +55,18 @@ class AMSWorkflow
    * execution */
   AMSPhysicFn AppCall;
 
+  /** @brief A string identifier describing the domain-model being solved. */
+  std::string domainName;
```

Comment:
Is this expected to be unique? Do we rely on the user?

Reply:
There is no requirement for this to be unique. In practice it should.

Reply:
The API exposed to the application does not allow to register multiple domainNames.